app/services/open_feature/providers/dev_cycle.py

```
@@ -0,0 +1,251 @@
+                    flag_key=devcycle_key,
+                    default_value=None,  
+                    evaluation_context=context
+                )
+                
+                if result is not None:  # Flag exists in DevCycle
+                    logger.debug(f"DevCycle flag {devcycle_key}: {result}")
+                    return result
+                else:  # Flag doesn't exist in DevCycle, use fallback
+                    logger.debug(f"DevCycle flag {devcycle_key} not found, using environment fallback")
+                    
+            except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)